src/content/cloud/external-resources.mdx

```diff
@@ -0,0 +1,224 @@
+---
+title: External Resources
+description: Configure Okteto's manifest to include external resources in your development environment
+sidebar_label: External Resources
+id: external-resources
+---
+
+import Image from '@theme/Image';
+
+The External Resources feature in Okteto is a powerful mechanism that enables you to manage the lifecycle of external resources as part of your Okteto development environment. This means you can define the external resources that comprise your application, include those resources in Okteto's garbage collection process, and even destroy those resources when you finish working with your development environment. 
+
+Here are just a few examples of external resources you can use with Okteto:
+
+- A markdown file (e.g. README, API Documentation)
+- MongoDB Atlas Instance
+- LaunchDarkly
+- Amazon SQS
+- Amazon S3
+- Amazon RDS databases
+- AWS Lambda Functions
+
+You no longer need to use different tools to work on your application in Okteto and you won't need to worry about rogue external resources continuing to run once you are done with your work.
+
+When you configure External Resources in Okteto you can see those resources within the Okteto UI, can click to go directly to those resources in their respective platform, and include them in your Okteto `deploy` and `destroy` configurations.
+
+## Overview
+
+When you configure the `external` section in your [Okteto manifest](/reference/manifest), keep in mind the following:
+
+- You can create a container image that includes any external tools (to Okteto) that you need in order to provision or access your external resources.
+- This container image can be included in your Okteto manifest's `deploy` and `destroy` sections
+- External resources are destroyed when your Okteto namespace is deleted
+- External resources are covered by Okteto's [garbage collection process](/administration/cleanup/#delete-unused-namespaces)
+
+## Configuring external resources
```

Comment:
The keys to access the resources also need to be set. Should we mention this here? (Or do you plan an admin-level counterpart for this?

This is a big value add: admin centralizes credentials instead if giving them to every developer

Reply:
@rberrelleza I hadn't planned a separate section. I think it makes sense to include here and we can consider splitting it later if we find that's the right move 😄 

For my edification: are you saying that an `admin` could (not a hard requirement) specify the `Secrets` in the `Admin` > `Secrets` page that then all developers would use for a given resource? 

So all namespaces that deploy an application that uses (for example) AWS S3 buckets would rely on the admin-set keys for an organization in AWS vs creating their own AWS accounts and managing those secrets locally on their own Okteto account?

Just want to calibrate so I know how to write this section 👍

Reply:
Exactly. , that's the way to go for anything that requires an API key.

For AWS and GCP specifically, we also support a "credential-less" model for self-hosted customers, where they can define security roles and use those to give permissions to Okteto instead of using AP Keys. This is the recommended way by both AWS and GCP (Provecho is giving a talk about this on KubeCon if you are interested).

Reply:
Latest commit includes some language to address this: https://github.com/okteto/docs/pull/363/commits/7f3d39ef99fe1ab10b55992f331083da4524d11b